proteus/FemTools.py

```diff
@@ -529,18 +521,26 @@ def __init__(self,nd=3, order=2):
         elif nd == 2:
             for j in range(order+1):
                 for i in range(order+1):
-                    basis.append(lambda xi,i=1,j=j:self.fun[i](xi[0])*self.fun[j](xi[1]))
+#                    basis.append(
+#                        lambda xi,i=1,j=j: self.fun[i](xi[0])*self.fun[j](xi[1]))
+#                    basisGradients.append(
+#                        lambda xi,i=i,j=j: numpy.array([self.dfun[i](xi[0])*self. fun[j](xi[1]),
+#                                                        self. fun[i](xi[0])*self.dfun[j](xi[1])]))
+#            funMap=[0,4,1,  7,8,5,   3,6,2]
```

Comment:
In the past we've left in a lot of commented out code like  this, but I'm trying to do a lot  less  of this sort of thing. I would only leave  it in if it's explanatory, as in, "here is one way to do it, but for now we're going with the following approach, which seems better because..."